cleaners/firefox.xml

```diff
@@ -18,51 +18,133 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app Mozilla Firefox
+    @url https://www.mozilla.org/en-US/firefox/new/
+    @os Windows, Linux, FreeBSD, OpenBSD
+    @cleanerversion v2.1.1
+    @cleanerdate 2019-03-29
+    @cleanerby Andrew Ziem (2018-10-01 - 2018-11-05) & luoliyan (2018-10-31) & danielk43 (2018-11-29) & Tobias B. Besemer (2019-02-12 - 2019-03-29)
+    @tested ok v67.0a1, Windows 7
+    @testeddate 2019-03-15 (... with an old version of the cleaner!)
+    @testedby https://github.com/Tobias-B-Besemer
+    @note The cleaner is for different editions, like the normal "Mozilla Firefox" edition and e.g. the "Mozilla Firefox Developer Edition"...
 
+-->
+<!--
+This cleaner includes first steps to make a universal "All Firefox based browsers"-Code-Base!
+I will fork all_firefox_based.xml later from this file.
+And then fork for each browser a file from all_firefox_based.xml.
+Tobias.
 -->
 <cleaner id="firefox">
   <label>Firefox</label>
   <description>Web browser</description>
   <running type="exe" os="unix">firefox</running>
   <running type="exe" os="unix">firefox-bin</running>
   <running type="exe" os="windows">firefox.exe</running>
+  <!--
+  Just a guess! I will figure this out later! Tobias.
+  <running type="exe" os="linux">firefox</running>
+  -->
+  <!-- If you already wanna give the universal cleaner a try: Remove the comment-marks from the following block... -->
+  <!-- The following values are just guesses... -->
+  <!--
+  <running type="exe" os="windows">seamonkey.exe</running>
+  <running type="exe" os="linux">seamonkey-bin</running>
+  -->
+  <!--
```

Comment:
The comments on 45-55 should be removed. If someone wanted to try ths with SeaMonkey, it would work without the `running` element, which only stops the cleaning from modifying files when the application (SeaMonkey) is running.

Reply:
K, will clean up!